extensions/cornerstone/src/services/SegmentationService/SegmentationService.ts

```diff
@@ -112,7 +123,7 @@ class SegmentationService extends PubSubService {
     toolGroupId?: string,
     properties?: {
       label?: string;
-      color?: Types.Point3;
+      color?: ohifTypes.RGB;
```

Comment:
Thank you - this will make it easier to find/update color handling int he future.